commands/initial/templates/tasks/test.js

```diff
@@ -13,45 +13,40 @@ function run(type) {
     server.start();
 }
 
-function getConfig(type) {
-    switch (type) {
-        case 'headless':
-        case 'hl':
-        case 'h':
-            return getHeadlessConfig();
-        case 'all':
-        case 'a':
-            return getAllConfig();
-        case 'watch':
-        case 'w':
-            return getWatchConfig();
-        default:
-            return getSingleConfig();
-    }
-}
-
-function getSingleConfig() {
-    let config = getHeadlessConfig();
-
-    config.singleRun = true;
+function getConfig(options) {
+	let config = getAllConfig(options.watch);
+	config.browsers = options.browsers;
+	config.singleRun = !options.watch;
 
-    return config;
+	return config;
 }
 
-function getHeadlessConfig() {
-    let config = getAllConfig();
-
-    config.browsers = ['PhantomJS'];
-
-    return config;
-}
-
-function getWatchConfig() {
-    let config = getAllConfig(true);
-
-    config.browsers = ['Chrome'];
-
-    return config;
+function parseOptions(args){
+	let options = {};
+
+	args.forEach((arg, index)=>{
+		let value = arg.match(/^(\w+)/)[1];
+		switch(value){	//there are probably libraries that parse cmd line arguments...
```

Comment:
there are, but when I started this, the intent was to keep it as simple & lightweight as possible, as you might see, there aren't a ton of external includes from Librarian code

Reply:
I wanted to mention, Librarian has its own focused utility for handling options:
https://github.com/gonzofish/angular-librarian/blob/master/tools/utilities/options.js

Here's an example of how its used:
https://github.com/gonzofish/angular-librarian/blob/master/commands/component/index.js#L20

Again, totally understand there are libraries that do this, but when I started Librarian I was just having fun and rolled my own--the `options.js` file is fully tested, though:
https://github.com/gonzofish/angular-librarian/blob/master/test/tools/utilities/options.spec.js

Reply:
Oh! I didn't realize you had code to handle that. The original code wasn't using it either and is the only reason I suggested finding a library. I'll see what I can do.

Reply:
Ok now I've tried to incorporate options.js and my inexperience with writing developer tools may be biting me in the butt. This file (test.js) gets moved to the host library while options.js stays in the angular-librarian folder. This gives it two different requires paths: one to allow the library to build ('../../../../tools/utilities') and one from the host library ('angular-librarian/tools/utilities'). Is there something in the initialization that will resolve this automatically or am I supposed to pretend that test.js is in a host library and use that path?

Reply:
This file, `test.js`, ends up as part of the library's code base, as an NPM script. So the proper thing to do would be to access it using `angular-librarian/tools/utilities` (or `angular-librarian/tools/utilities/options`)

Reply:
Now I've attempted to use the `options.js` file in `test.js` and I have some reservations about it. I've updated the code to conform to the style guide lines you've given me. Is using the options code a deal breaker? Because I'd like to ask a few questions about that separately.